src/MCPClient/lib/clientScripts/upload-archivesspace.py

```diff
@@ -123,13 +123,33 @@ def upload_to_archivesspace(files, client, xlink_show, xlink_actuate, object_typ
 
         # Get file & format info
         # Client wants access copy info
+
+        original_name = ''
+        # Get file & format info
+        # Client wants access copy info
+        try:
+            original_file = File.objects.get(
+                filegrpuse='original',
+                uuid=uuid
+            )
+        except (File.DoesNotExist, File.MultipleObjectsReturned):
+            original_name=''
+            size = format_name = format_version = None
+        else:
+            #set some variables based on the original, we will override most ofthese
```

Comment:
Minor nitpick, but comments should have a space between the hash and teh text. (It's worth configuring a linter, it'll catch stuff like this.)